src/finishRes.ts

```diff
@@ -42,10 +43,37 @@ export function finishRes(res: ValidatableResponse, status: number, json: Format
   if (isErrorsResponse(json)) {
     json.errors.forEach(error => {
```

Comment:
give how often you need to cast to "any" below, I'd recommend either:

1. Changing the ErrorsResponse type above to include the "data" error property:
```
interface ErrorWithData extends Error {
  data: {
    // ...
  }
}

interface ErrorsResponse {
  message: string;
  errors: ErrorWithData[];
}
```

2. Just cast to any in the forEach callback signature:

```
json.errors.forEach((error: any) => {
```